examples/aurora.ipynb

```diff
@@ -480,7 +480,7 @@
     "        )\n",
     "\n",
     "    elif iteration % 2 == 0:\n",
-    "        repertoire, previous_error = aurora.container_size_control(\n",
+    "        repertoire, previous_error = jax.jit(aurora.container_size_control)(\n",
```

Comment:
I think when you do `aurora.container_size_control`, Python considers it as a function with a different id. So `jax.jit(aurora.container_size_control)` keeps rejitting the function over and over again.

The solution in this case consists of doing what you did in some tests: setting a fixed function `container_size_control_fn = jax.jit(aurora.container_size_control)` and then only use `container_size_control_fn`.